xml_converter/generators/cpp_templates/attribute_template.hpp

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include "../rapid_helpers.hpp"
+#include "../rapidxml-1.13/rapidxml.hpp"
+#include "../string_helper.hpp"
+
+using namespace std;
+
+class {{class_name}} {
+ public:
+		{%- for attribute_variable in attribute_variables: %}
```

Comment:
They get removed with `{%-` but you are using tabs as indentation on this line while using spaces on all other lines.